.github/ISSUE_TEMPLATE/Bug.md

```diff
@@ -5,26 +5,49 @@ about: Report a problem with rclone
 
 <!--
 
-Welcome :-) We understand you are having a problem with rclone; we want to help you with that!
+We understand you are having a problem with rclone; we want to help you with that!
 
-If you've just got a question or aren't sure if you've found a bug then please use the rclone forum:
+**STOP and READ**
+**YOUR POST WILL BE REMOVED IF IT IS LOW QUALITY**:
+Please show the effort you've put in to solving the problem and please be specific.
+People are volunteering their time to help! Low effort posts are not likely to get good answers!
+
+If you think you might have found a bug, try to replicate it with the latest release or latest beta.
```

Comment:
At a second glance, should we leave it to the user for deciding to switch to either the latest stable or beta version to try?

And also, we should probably indicate that the operation cannot be reversed (at least not without noting down the previous version, which sounds like a good idea to print while doing the update).

Reply:
> At a second glance, should we leave it to the user for deciding to switch to either the latest stable or beta version to try?
The text does leave it to the user. Should I replace "latest release" with "latest `stable` release"?

> And also, we should probably indicate that the operation cannot be reversed.

Add `You might want to backup your current rclone executable before the update` before `If your rclone reports `unknown command "self-update"` then you'll need to update manually following the install instructions at https://rclone.org/install`
??

Reply:
> The text does leave it to the user

I meant should we leave it or not? Some issues/features may only be fixed/added in the latest beta 
with a point release pending.

> You might want to backup your current rclone executable before the update

What about deb/rpm packages? Ideally, this should be done by self-update where it prints the version before going for the update so that a revert is as easy as specifying that version with the self-update command.

Reply:
@darthShadow 

> I meant should we leave it or not? Some issues/features may only be fixed/added in the latest beta
> with a point release pending.

Let's look at the context around this template.

There are two groups of users submitting bug reports:
1. Home/office rclone users ("normal" users)
2. Developers who understand rclone internals, maybe from the rclone team

Professionals will most probably know for sure what they want. The template is targeted at an average user.

The user will see this template when he/she submits a new bug request on github.
They can come to this point  through three ways:
1. They found a bug in rclone all alone
2. They come from the forum after a number of discussions with rclone support
3. They come from another github issue after discussions with rclone developers or other users

Users coming through ways 2 and 3 already have some background on the issue they report.
They know when and where the problem manifests itself.
Probably they had been given a prior advice on what version to try and they will follow it rather than a generic boilerplate.
We are targeting the way no.1 here.

We have the following options:
1. Leave user a _choice_ between latest stable and beta - (as it is in the current template)
2. Keep only _latest stable_ - (as it generally was before this pull request, I will elaborate below)
3. Keep only _latest beta_ - (this is what you want, let's elaborate below)
4. Advice on a _concrete beta_ version - (inappropriate for a generic template)
5. There will never be a "concrete stable option", there will be only "last stable"

No.5 is meaningless.
No.4 is for advanced users coming from forum or from another bug thread on github.

@ncw **Please help! Choosing between 1,2,3 needs more discussion. I can't decide without you!**

> What about deb/rpm packages?

I don't want to make a generic bug template too complex or convert it into a _detailed_ manual page for the selfupdate command. We already have a link from the template to the quick start in https://rclone.org/install which hides install details from the template. Currently even that page bypasses deb/rpm details and links further to the self-update html page for those.

If you want to bring deb/rpm details two levels up right into the bug template, there should be a reason for that.
Currently the deb/rpm package contains a single binary, a MIT license (it never changes) and a manual page. The selfupdate command without the deb/rpm flag will still change the binary _so the user can reproduce their bug_. It will probably leave the local manual out of sync. The self-update command should warn about it. The self-update help page should warn about it. But it's not a duty of the bug template. Its duty is to tell user how to _reproduce and report a bug_.

My goal for this command was to make things simple for end user.
The command hides gory details from user - they don't need to open download.rclone.org in the browser, save archive in a temporary place, unpack, apply permissions. You just type a single command and hit enter.
The command simplifies replies of support - instead of "get the https://xxx.rclone.org/long_url" they can write "please run rclone selfupdate --beta --version xxx".

By adding more details here, by piling up features we lose the very goal of simplicity.
Compare with "restic selfupdate" command which has only one option: "--output PATH" for several releases.
The "rclone selfupdate" has 5 options from the very start. I think it's a wrong direction.

> Ideally, this (backup) should be done by self-update where it prints the version before going for the update so that a revert is as easy as specifying that version with the self-update command.

So the `rclone self-update` will either print `Successfully updated from OLDVER to NEWVER`
(or e.g. `Successfully updated to NEWVER. Use "rclone selfupdate [--beta] --version OLDVER" to restore`).
It's a _good point_ deserving a ticket on its own.  Let's not mix it in this thread or we lose both.

cc @ncw

Reply:
Alright, then we can probably skip the update command altogether and redirect them to a page on the website with the usage details of self-update. That page will/can contain all the details of how to upgrade and how to revert between stable/beta versions (depending on the choice between 1,2 & 3). This can probably be something as simple as (once we decide on either stable or beta or letting the user decide):

```
If you think you might have found a bug, try to replicate it with the latest beta (or stable). The upgrade instructions are available at https://rclone.org/commands/rclone_selfupdate/#beta (or stable)
```

This will simplify the template and have only a single point of reference everywhere rather than duplicating it across templates.

Reply:
You are right. :-)
The target page can be seen here until the next release (of course, I will commit without the `tip` part) :
https://tip.rclone.org/commands/rclone_selfupdate/

I mentions deb/rpm packages but lacks details about beta updates in the synopsis section.
It does not explain how to revert either. I will improve its text shortly!
Unfortunately this backlog PR has a special lifecycle, so I will have to do that in a separate PR and ask you to review!

Reply:
The synopsis part in https://tip.rclone.org/commands/rclone_selfupdate/
should also contain the following clause (I will remove from the Feature template):
```
This command was not available before rclone version 1.55, if your rclone reports
`unknown command "self-update"` then you'll need to update manually
following the install instructions at https://rclone.org/install/
```
Thus we gather all update details in a single point.